src/commands/clan/brains.ts

```diff
@@ -44,7 +43,7 @@ export async function execute(interaction: CommandInteraction) {
         description:
           "Captain Scotch, kenny kamAKAzi, and 3BH can pilot dread multis for any class of brain, subject to multi restrictions.",
         fields: BASE_CLASSES.map((playerClass) => ({
-          name: `**__${playerClass}__**`,
+          name: bold(underscore(playerClass)),
```

Comment:
while i think the squigglies are mostly fine i will readily admit this is annoying to read and bold(underscore( scans better lol

Reply:
It just makes it less error prone too. No chance of failing to match it on the other end etc.